src/Geocoders/CachingGeocoder.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Maps\Geocoders;
+
+use DataValues\Geo\Values\LatLongValue;
+
+/**
+ * @since 5.0
+ *
+ * @licence GNU GPL v2+
+ * @author HgO < hgo@batato.be >
+ */
+class CachingGeocoder implements Geocoder {
+	protected $geocoder;
+	
+	protected $cache;
```

Comment:
The MediaWiki style of having thins be protected by default, rather than private by default, is harmful in my opinion.